src/actions/index.js

```diff
@@ -1,67 +1,106 @@
-import axios from 'axios';
+import axios from "axios";
 
-import { FETCH_CLUBS, GET_PAGES, GET_HEROS, GET_EVENTS, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
+import {
+  FETCH_CLUBS,
+  GET_PAGES,
+  GET_HEROS,
+  GET_EVENTS,
+  MODAL_OPEN,
+  MODAL_CLOSE,
+} from "./actionTypes";
 
-const api = 'https://dev.huoa.org/wp-json';
+const api = "https://dev.huoa.org/wp-json";
+let isProd =
+  process.env.NODE_ENV === "production" && navigator.userAgent !== "ReactSnap";
```

Comment:
is this a prettier style? if it is could you wrap in parentheses so its a bit more explicit about the new line and where the bool ends?

Reply:
Yeah, prettier does that! I put it back on one line and disabled prettier.